src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -476,6 +490,7 @@ union uct_ib_mlx5_hca_cap_union_bits {
     struct uct_ib_mlx5_cmd_hca_cap_bits cmd_hca_cap;
     struct uct_ib_mlx5_odp_cap_bits odp_cap;
     struct uct_ib_mlx5_atomic_caps_bits atomic_caps;
+    struct uct_ib_mlx5_cmd_hca_cap_2_bits cmd_hca_cap_2;
```

Comment:
what does `2` mean here?

Reply:
@evgeny-leksikov The name of cap_2 comes from the PRM. In the PRM some caps are defined as hca_cap, some caps are defined as hca_cap_2.
```
62.Added HCA_CAP 2. See Section 29.3.4.2, “HCA Device Capabilities 2”, on page 2071.
```